cmd/main.go

```diff
@@ -50,17 +55,31 @@ func action(cliCtx *cli.Context) error {
 
 	logger.MainLog.Infoln("NEF version: ", version.GetVersion())
 
+	ctx, cancel := context.WithCancel(context.Background())
+	sigCh := make(chan os.Signal, 1)
+	signal.Notify(sigCh, os.Interrupt, syscall.SIGTERM)
+	wg := new(sync.WaitGroup)
+	wg.Add(1)
```

Comment:
I think the use of wg here is useless, the reasons listed below:
- The wg wasn't passed to any goroutines
- No blocking operation invoked (`wg.Wait()`)